correios/models/posting.py

```diff
@@ -64,6 +64,8 @@ def __init__(self,
                  state: str = "",
                  document: str = "",
                  comment: str = "",
+                 description: str = "",
+                 details: str = "",
                  ):
```

Comment:
It was my first experience developing with type annotations. It's... weird as a coding feature. But, believe me, it's the best documentation tools that I've used together with good identifier names and clean code :)

And it make life awesome for Rich-IDEs users :) But I'll not use it anymore.